src/App.tsx

```diff
@@ -17,18 +68,34 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                filterByStatus={filterByStatus}
+                filterByQuery={filterByQuery}
+                query={query}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {todos.length
```

Comment:
Better to compare explicitly, check why:
https://mate-academy.github.io/style-guides/javascript.html#153use-shortcuts-for-booleans-but-explicit-comparisons-for-strings-and-numbers
```suggestion
              {todos.length !== 0
```